### PR TITLE
DROOLS-3324 Update properties in persistence.xml

### PR DESCRIPTION
**Pull request by baldimir**
@jiripetrlik @mariofusco could you please review? The properties needs to be updated because of Nayarana changes. Without them the tests fail on DB2 database randomly.

### CODE REVIEW

**Review by rsynek — APPROVED** [review]
Looks good, the same property has been used for jBPM

File: drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -21,8 +21,12 @@
         <property name="hibernate.dialect" value="${maven.hibernate.dialect}"/>
         <property name="hibernate.default_schema" value="${maven.jdbc.schema}"/>
          
+        <!-- BZ 841786: AS7/EAP 6/Hib 4 uses new (sequence) generators which seem to cause problems -->
```
Comment (MarianMacik) [inline drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml:24]: Maybe add `+` after EAP 6 and Hib 4 so somebody else doesn't think that it is no longer needed.

**Review by MarianMacik — APPROVED** [review]
I would just improve the comment, otherwise looks good.

File: drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml (hunk 1 above)
Comment (baldimir) [reply drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml:24]: It is the same comment as in jBPM.

**Review by jiripetrlik — APPROVED** [review]
Looks good to me.

File: drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml (hunk 1 above)
Comment (MarianMacik) [reply drools-persistence/drools-persistence-jpa/src/test/filtered-resources/META-INF/persistence.xml:24]: Then fine by me.

**Pull request merged**